src/cbmc/bmc.h

```diff
@@ -147,7 +148,8 @@ class bmct:public safety_checkert
     message_handlert &_message_handler,
```

Comment:
Add doc for the `callback_after_symex` specifically what the return value of the callback should indicate